src/ship/PlayerShipController.cpp

```diff
@@ -166,6 +166,13 @@ REGISTER_INPUT_BINDING(PlayerShipController)
 	auto landingGroup = controlsPage->GetBindingGroup("LandingControl");
 	input->AddActionBinding("BindToggleLandingGear", landingGroup, Action({ SDLK_n }));
 	input->AddAxisBinding("BindControlLandingGear", landingGroup, Axis());
+
+	auto radarGroup = controlsPage->GetBindingGroup("RadarControl");
```

Comment:
This should likely be in a new "Flight HUD" page rather than "Ship - Controls" - the latter is intended for bindings affecting the physical control state of the ship, while the new keybindings are related only to the radar widget and don't affect anything about the "in-universe" ship.

Reply:
Ok; this may take a few days (have a conference the next 2 days, then hopefully away on a motorbike trip)

Reply:
Fixed in e2b015d534b99b2a9776d7fa669a565bc2b782ae